llvm/lib/CodeGen/SelectionDAG/TargetLowering.cpp

```diff
@@ -8177,6 +8177,64 @@ SDValue TargetLowering::expandFMINNUM_FMAXNUM(SDNode *Node,
   return SDValue();
 }
 
+SDValue TargetLowering::expandFMINIMUM_FMAXIMUM(SDNode *N,
```

Comment:
Could you please add one or both of:
- some comments here showing what MIR is generated, and/or
- some MIR tests with -run-pass=legalizer showing the expansion with various combinations of fast math flags.

Reply:
Can MIR be valid input for legalizers? I see MIR outputs are all legalized.

Reply:
> Can MIR be valid input for legalizers?

No, sorry, please ignore that part. I was thinking about GlobalISel not SelectionDAG.